frame/system/src/lib.rs

```diff
@@ -262,9 +262,9 @@ pub mod pallet {
 	#[pallet::hooks]
 	impl<T: Config> Hooks<BlockNumberFor<T>> for Pallet<T> {
 		fn on_runtime_upgrade() -> frame_support::weights::Weight {
-			if !UpgradedToDualRefCount::<T>::get() {
-				UpgradedToDualRefCount::<T>::put(true);
-				migrations::migrate_to_dual_ref_count::<T>()
+			if !UpgradedToTripleRefCount::<T>::get() {
+				UpgradedToTripleRefCount::<T>::put(true);
+				migrations::migrate_to_triple_ref_count::<T>()
```

Comment:
Can we not call the migration explicitly in the particular runtimes?

Reply:
sure - i'm just concerned about it being forgotten.